### DISCUSSION TITLE
Who is using React Native for Web?

### DISCUSSION THREAD

**Discussion by necolas** [necolas/react-native-web#2417] [Share]
*Please only comment to share how you are using React Native for Web*.

Include the company name, link to its website, and links to any OSS repositories using react-native-web.

**Comment by yuxizhe** [comment]
We use react-native-web in Xueqiu (雪球), or Snowball Finance https://xueqiu.com, a FinTech company with over 12 million registered monthly active users . Thanks to React Native and React Native Web, In 2022, 70% of our new features are wroten in  the same code  for our iOS app / Android App / mobile web (H5).
<p align=center><img src="https://user-images.githubusercontent.com/14990734/203590635-032754a4-1b8a-496f-91c6-25c79ae6ad8e.png" alt="图片" width="50%" /></p>


React-native-web  improves our development efficiency, we can write&debug  UI / react states / server api in Chrome ,  which is much more convenient than using phone or Simulator.

We wrote articles to introduce the advantage of React-native-web, and our practical experience in Xueqiu. for example : https://juejin.cn/post/7168778901970223111

We alse build a little universal components library for iOS / Android / mobile web with styled-system https://snowbox.js.org/
<p align=center><img src="https://user-images.githubusercontent.com/14990734/203591893-a224f9be-981f-48ef-9e35-f15765bdb8dc.png" alt="图片" width="50%" /></p>

**Comment by thematheusmello** [comment]
Thank you @yuxizhe 
It is my first time using this library, and I am considering using it but is not clear to me how to structure the project.
I am curious if companies like the ones that you mentioned or even Twitter are using the same screens, components, utils, and so on for both repositories or just the components and utils but later on, they are building the pages/screens, navigations and other things like that per repository.

This is more of a conceptual question about structuring the project and I would like to hear from you.
Thank you in advance.

**Reply by necolas** [reply]
Please open a new discussion to ask about this. I'd like to keep this thread focused on the topic. Thanks
